api/v1alpha1/virtualmachine_types.go

```diff
@@ -112,6 +112,27 @@ const (
 	FirstBootDoneAnnotation = "virtualmachine." + GroupName + "/first-boot-done"
 )
 
+// VirtualMachine backup/restore related constants.
+const (
+	// ManagedByExtensionKey and ManagedByExtensionType represent the ManagedBy field on the VM.
+	// Historically, this field was used to differentiate VM Service managed VMs from traditional ones.
```

Comment:
by adding `historically`, does it mean this field is no longer used to differentiate?

Reply:
I am not sure if this is used in any other context besides backup/restore before. The comments are copied from the previous place.

Reply:
Added an additional comment to clarify in the latest commit.

Reply:
Updated the comment of these fields for better clarity.